### DISCUSSION TITLE
part-ib/2019-77/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#69] [Question comments]
Part IB 2019 Variational Principles: Paper 1, Section I, A 

https://questions.tripos.org/part-ib/2019-77/

**Comment by aleggs0** [comment]
As far as I know, this is the most difficult tripos question in recent history. (/j)

**Comment by Mohan-Mo** [comment]
I got just (0,0,0) and (42, 12, -6) using the Lagrangian.

**Reply by Mohan-Mo** [reply]
ah nvm lol
